TERMINAL LIST
Type:      File
Name:      f1
Original:  <dir>/r3/f1
Created:   2020-05-31 12:00:00 
Entries:      1
Total Files:  1
Log size:     0.00B
Size:         10.00B 


EXCEPTION RESTORE
Encountered exception(s)

- Path not found: bad file



CallStack (from HasCallStack):
  throwCS, called at src/SafeRm.hs:269:5 in safe-rm-0.1-inplace:SafeRm

LOGS RESTORE
[2020-05-31 12:00:00][functional.restore][Debug] Trash home: <dir>/r3/.trash
[2020-05-31 12:00:00][functional.restore.readIndex][Debug] Index path: <dir>/r3/.trash/.index.csv
[2020-05-31 12:00:00][functional.restore.readIndex.readIndexWithFold][Debug] Found: MkPathData {pathType = PathTypeFile, fileName = MkPathI {unPathI = "f1"}, originalPath = MkPathI {unPathI = "<dir>/r3/f1"}, created = MkTimestamp {unTimestamp = 2020-05-31 12:00:00}}
[2020-05-31 12:00:00][functional.restore][Info] Restored: 
[2020-05-31 12:00:00][functional][Error] Encountered exception(s)

- Path not found: bad file



CallStack (from HasCallStack):
  throwCS, called at src/SafeRm.hs:269:5 in safe-rm-0.1-inplace:SafeRm


